nni/retiarii/experiment/pytorch.py

```diff
@@ -193,6 +193,8 @@ def _start_strategy(self):
         )
 
         _logger.info('Start strategy...')
+        search_space = BaseStrategy.get_model_space(base_model_ir, self.applied_mutators)
```

Comment:
Should call `dry_run_for_formated_search_space` here. Not sure whether every search space can be successfully formatted though.